docs/source/gettingstarted.rst

```diff
@@ -3,15 +3,53 @@
 Installing Mantarray File Manager
 ==================================
 
-This section will demonstrate how to install the package and get started with it.
+We recommend using Jupyter Notebooks to install the mantarray-file-manager. This
+section will walk you through the process of using Jupyter notebooks and installing the
+package.
 
-Download mantarray_file_manager
---------------------------------
 
- * If you have already installed PIP and Python on your device, then run the command "pip install mantarray_file_manager"
- * If you haven't installed Python proceed to the following URL: https://www.python.org/downloads/
- * If you haven't installed PIP run in your command line (WINDOWS): "python get-pip.py"
- * If you haven't installed PIP run in your terminal (MAC): "sudo easy_install pip"
+First thing first, is to make sure you have python dowloaded to your PC. To do so,
+proceed to the following website to download it: https://www.python.org/downloads/
 
-Working With mantarray_file_manager
-------------------------------------
+As you are downloading this, make sure to select the options for installing pip
+and on the next page the option for "Add Python to Environment Variables"
+
+Here are images below to show what should be selected in full:
+
+.. figure:: /images/python_install.PNG
+.. figure:: /images/python_install_2.PNG
+
+
+Once python has been installed we will now install Jupyter for Jupyter Notebooks.
+
+To do so on Windows type "cmd" into the search on the taskbar to open up the command
+terminal. From here you will want to follow these commands:
+
+ * First you want to make sure your pip is up to date: python -m pip install --upgrade pip
```

Comment:
please use some sort of annotation to make the code appear differently.  `like this` (in markdown...I'm sure there's an equivalent in RST)

Reply:
(also command line commands)